### PR TITLE
Goal component

### PR DESCRIPTION
**Pull request by skuhlmann**

### CODE REVIEW

File: app/components/user-goal.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({});
```
Comment (trayo) [inline app/components/user-goal.js]: What does a component do?

**Comment by skuhlmann** [comment]
A component is like a partial on acid.

**Comment by skuhlmann** [comment]
I mean a partial on steroids.

**Comment by chandracarney** [comment]
Everything looks good for the styling and JS

**Pull request merged**